src/domain/request/usecase/ExecuteReadyRequestsUseCase.ts

```diff
@@ -56,11 +55,13 @@ class ExecuteReadyRequestsUseCase {
         request.getSelectionPath(),
       );
       response.addSelectedData(selectedData);
+      this.logger.info(`Sending response [response=${JSON.stringify(response)}]`);
```

Comment:
This method (`fetchAndSelectData`) doesn't know anything about sending responses. You should rename logs content.

Reply:
Any specific idea? I just moved this line, didn't write so not sure what was intention of an author.

Reply:
You're in a `fetchAndSelectData` function so how about 
```suggestion
      this.logger.info(`Fetched and selected data [response=${JSON.stringify(response)}]`);
```